inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convert_color_ref.cl

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "include/batch_headers/fetch_data.cl"
+#include "include/batch_headers/data_types.cl"
+
+#ifdef CONVERT_FROM_NV12
+#ifdef BUFFER_MEM
+KERNEL(convert_color_ref)(const __global INPUT0_TYPE* input_y,
+#if INPUTS_COUNT == 2
+                          const __global INPUT1_TYPE* input_uv,
+#endif
+                          __global OUTPUT_TYPE* output) {
+
+    const uint b = get_global_id(0);
+    const uint f = get_global_id(1);
```

Comment:
I suppose currently you don't use this dimension. Maybe it's better to use gws = { x, y, b }?
If get_global_size(1) > 1 then you have several parallel threads which make one job.